app/templates/_gulpfile.js

```diff
@@ -10,10 +10,16 @@ var options = {
   dist: '<%= props.paths.dist %>',
   tmp: '<%= props.paths.tmp %>',
   e2e: '<%= props.paths.e2e %>',
-  errorHandler: function(title) {
+  failOnLintErrors: true,
+  failOnCompileErrors: true,
+  errorHandler: function(title, throw_) {
```

Comment:
throw is a reserved keyword
Le 20 févr. 2015 11:33, "Mehdy Dara" <notifications@github.com> a écrit :

> In app/templates/_gulpfile.js
> <https://github.com/Swiip/generator-gulp-angular/pull/279#discussion_r25061614>
> :
>
> > @@ -10,10 +10,16 @@ var options = {
> >    dist: '<%= props.paths.dist %>',
> >    tmp: '<%= props.paths.tmp %>',
> >    e2e: '<%= props.paths.e2e %>',
> > -  errorHandler: function(title) {
> > +  failOnLintErrors: true,
> > +  failOnCompileErrors: true,
> > +  errorHandler: function(title, throw_) {
>
> Why throw_ ? strange name for a variable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Swiip/generator-gulp-angular/pull/279/files#r25061614>
> .
>